### PR TITLE
feat(508): Text alternatives for non-text content

### PR DESCRIPTION
**Pull request by SOAP1914**
@wski  
@clarkjefcoat

### CODE REVIEW

File: app/js/components/createEdit/form/ImageInput.jsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ var ImageInput = React.createClass({
                     <button onClick={this.removeImage} className="btn image-remove">
                         <i className="icon-trash" />
                         </button>
-                    <img ref="image" className="image-preview"
+                    <img alt="" ref="image" className="image-preview"
```
Comment (wski) [inline app/js/components/createEdit/form/ImageInput.jsx]: Is this supposed to have text in the alt tag?

File: app/js/components/management/mall/Intents.jsx [hunk 2]
```diff
@@ -107,7 +107,7 @@ var Intents = React.createClass({
                     { field: 'icon', caption: 'Icon', size: '50px',
                         render: function (record) {
                             return record.icon ?
-                                `<img style="width:24px;height:24px;" src=${record.icon} >` :
+                                `<img style="width:24px;height:24px;" alt='record icon' src=${record.icon} >` :
```
Comment (wski) [inline app/js/components/management/mall/Intents.jsx]: We should make this a bit more descriptive, to explain to a user what the icon does when you click it, not so much what it's a picture of

File: app/js/components/createEdit/form/ImageInput.jsx (hunk 1 above)
Comment (SOAP1914) [inline app/js/components/createEdit/form/ImageInput.jsx]: yes it left blank so jaw can skip over it

File: app/js/components/management/mall/Intents.jsx (hunk 2 above)
Comment (SOAP1914) [inline app/js/components/management/mall/Intents.jsx]: what does the icon do, I didn't now I assume it record idk

File: app/js/components/discovery/FeaturedListingTile.jsx [hunk 3]
```diff
@@ -21,7 +21,7 @@ var FeaturedListingTile = React.createClass({
         return (
             <li className="listing SearchListingTile SearchListingTile--featured">
                 <a href={href}>
-                    <img src={image} />
+                    <img alt=`Launch ${listing.title} application` src={image} />
```
Comment (wski) [inline app/js/components/discovery/FeaturedListingTile.jsx]: ``

File: app/js/components/listing/ListingTile.jsx [hunk 4]
```diff
@@ -168,7 +168,7 @@ var AdminOwnerListingTile = React.createClass({
             <li className={classSet}>
                 <ActionMenu listing={listing} />
                 <a href={overview}>
-                    <img className="AdminOwnerListingTile__img" src={listing.imageLargeUrl} />
+                    <img alt=`Click to manage ${listing.title}` className="AdminOwnerListingTile__img" src={listing.imageLargeUrl} />
```
Comment (wski) [inline app/js/components/listing/ListingTile.jsx]: needs `{}`

> alt={`Click to manage ${listing.title}`}

File: app/js/components/quickview/Header.jsx [hunk 5]
```diff
@@ -31,7 +31,7 @@ var QuickviewHeader = React.createClass({
             <div className="quickview-header">
                 <div className="quickview-header-info">
                     <button type="button" className="close" aria-hidden="true" onClick={ this.props.onCancel }><i className="icon-cross-16-grayDark"></i></button>
-                    <img className="listing-icon" src={ image } data-fallback="/store/images/types/3" />
+                    <img className="listing-icon" alt=`${listing.title} header information` src={ image } data-fallback="/store/images/types/3" />
```
Comment (wski) [inline app/js/components/quickview/Header.jsx]: needs `{}`

> alt={`${listing.title} header information`}

File: app/js/components/createEdit/form/ImageInput.jsx (hunk 1 above)
Comment (wski) [inline app/js/components/createEdit/form/ImageInput.jsx]: Oh gotcha, cool

**Comment by arw180** [comment]
:+1:

**Comment by clarkjefcoat** [comment]
As soon as Ricardo fixes the Intents icon alt text - :+1:

**Comment by wski** [comment]
:+1:

**Comment by bhalt** [comment]
can this branch be deleted @SOAP1914?

**Comment by SOAP1914** [comment]
yes

**Pull request merged**